### DISCUSSION TITLE
Question about managing different elmish loops

### DISCUSSION THREAD

**Discussion by Jamie-Rower** [JordanMarr/ReactiveElmish.Avalonia#44] [General]
The App.fs file defines a global dispatch that other loops can access like app.Dispatch(SetView CounterView), which is awesome.  Is it possible for that to flow both ways?

If we have alerts delivered via message to the app loop, say, and we have a view running its own loop, would something like view.Dispatch(...) be viable?

I'm pretty sure this can be solved via bindings, but just letting my mind wander a bit and curious about a more informed view on the thought.

**Comment by JordanMarr** [comment]
You’re right that communication going the other way (app to view) would usually happen by the view creating a binding to app.

**Reply by Jamie-Rower** [reply]
yeah, I was thinking of it more as a means of bundling complex bindings, but I could think of a couple of reasons that may not work
